MobalMobal/MobalMobal/Signup/SignupCustomView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SignUpCustomView.swift
+//  MobalMobal
+//
+//  Created by jeongminho on 2021/02/28.
+//
+
+import SnapKit
+import UIKit
+
+class SignupCustomView: UIView {
+    let imageName: String
+    let inputText: String
+    
+    let radiusInputView: UIView = {
+        let view: UIView = UIView()
+        return view
+    }()
+    
+    lazy var imageView: UIImageView = {
+        let imageView: UIImageView = UIImageView(image: UIImage(named: imageName))
+//        imageView.frame.size = CGSize(width: 44, height: 44)
+        return imageView
+    }()
+    
+    lazy var textFieldView: UITextField = {
+        let textField: UITextField = UITextField()
+        textField.font = UIFont(name: "SpoqaHanSansNeo-Bold", size: 15)
+        textField.textColor = .textColor
+        textField.textColor = .white
+        textField.placeholder = inputText
+        return textField
+    }()
+    
+    init(imageName: String, inputText: String) {
+        self.imageName = imageName
+        self.inputText = inputText
+        super.init(frame: .zero)
+        
+        self.addSubview(radiusInputView)
+        radiusInputView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        self.radiusInputView.addSubview(imageView)
+        self.radiusInputView.addSubview(textFieldView)
+        
+        self.imageView.snp.makeConstraints { make in
+            make.centerY.equalTo(self.radiusInputView)
+            make.leading.equalTo(self.radiusInputView).offset(8)
```

Comment:
radiusInputView는 imageView의 superview니까
```suggestion
            make.centerY.equalToSuperview()
            make.leading.equalToSuperview().offset(8)
```
처럼 쓸 수도 있을 것 같아요!

Reply:
고렇네여 감사합니다